akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1349,8 +1337,8 @@ private[cluster] final class FirstSeedNodeProcess(seedNodes: immutable.IndexedSe
  *
  */
 private[cluster] final class JoinSeedNodeProcess(seedNodes: immutable.IndexedSeq[Address]) extends Actor with ActorLogging {
-  import InternalClusterAction._
   import ClusterUserAction.JoinTo
+  import InternalClusterAction._
```

Comment:
Yeah, I'll do that, there were loads of unused imports though, that's the reason I did it in the first place.